crates/stages/src/stages/hashing_account.rs

```diff
@@ -168,27 +130,37 @@ impl<DB: Database> Stage<DB> for AccountHashingStage {
         }
         let (from_block, to_block) = range.into_inner();
 
+        let stage_checkpoint =
+            input.checkpoint.and_then(|checkpoint| checkpoint.account_hashing_stage_checkpoint());
+
+        if stage_checkpoint.is_some() {
+            debug!(target: "sync::stages::account_hashing::exec", checkpoint = ?stage_checkpoint, "Continuing inner account hashing checkpoint");
```

Comment:
this debug log is valid only if the checkpoint range didn't change. pls move it below where we check that logic